test/framework/setup.go

```diff
@@ -525,7 +536,8 @@ networking:
 %s
 %s
 %s
-`, mesh, "80", "8080", serviceAddress, opts.serviceName, opts.protocol, opts.serviceVersion, opts.serviceInstance, serviceProbe, transparentProxy, opts.appendDataplaneConfig)
+%s
+`, mesh, "80", "8080", serviceAddress, opts.serviceName, opts.protocol, opts.serviceVersion, opts.serviceInstance, additionalTags, serviceProbe, transparentProxy, opts.appendDataplaneConfig)
```

Comment:
I don't want to block a PR because of this, but it got out of hand. Can we fallback to go template here? This can be a follow-up PR, issue, or whatever.

Reply:
https://github.com/kumahq/kuma/issues/8180